main/pipeline/pipeline.cc

```diff
@@ -1157,7 +1158,8 @@ incrementalResolve(core::GlobalState &gs, vector<ast::ParsedFile> what,
 
             auto canceled = runIncrementalNamer
                                 ? sorbet::namer::Namer::runIncremental(gs, absl::Span<ast::ParsedFile>(what),
```

Comment:
This is a non-blocking curiosity, not sure why it's written this way, but we don't need the `sorbet::` prefix on these calls.